### PR TITLE
box2d, fix installation paths

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
This was a fix to see if caveexpress would find Box2D files, from looking into the files they seem to be OK here, but caveexpress still doesn't seem to pick it up ...

**Comment by Begasus** [comment]
@korli could you check, looking at the cmake config files they seem to point to the correct paths for the headers and library

File: dev-games/box2d/box2d-2.3.1.recipe [hunk 1]
```diff
@@ -35,6 +36,7 @@ BUILD_REQUIRES="
 	devel:libGLU$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
+	cmd:ccache
```
Comment (korli) [inline dev-games/box2d/box2d-2.3.1.recipe]: Really?

**Comment by korli** [comment]
Box2D is used by a few recipes. Do they still build?

File: dev-games/box2d/box2d-2.3.1.recipe (hunk 1 above)
Comment (Begasus) [reply dev-games/box2d/box2d-2.3.1.recipe]: Needs to go, but in the process of "rebuilding" it's nice to have around :)

**Comment by Begasus** [comment]
> Box2D is used by a few recipes. Do they still build?

Need to check a few, but so far being used doesn't mean being "detected" ...

File: dev-games/box2d/box2d-2.3.1.recipe (hunk 1 above)
Comment (Begasus) [reply dev-games/box2d/box2d-2.3.1.recipe]: removed :)

**Comment by Begasus** [comment]
Tried out with Terrarium (0.1.9), with the previous version it doesn't show the version for box2d, with this one it shows version as >= 2.3.0
box2d

**Pull request merged**